tests/test_cli_compile.py

```diff
@@ -691,8 +691,7 @@ def test_annotate_option(pip_conf, runner, option, expected):
 
 
 @pytest.mark.parametrize(
-    "option, expected",
-    [("--allow-unsafe", "\nsetuptools=="), (None, "\n# setuptools==")],
+    "option, expected", [("--allow-unsafe", "\nsetuptools=="), (None, "\n# setuptools")]
```

Comment:
Also, these two changes are even in the same commit.

Reply:
What do you mean? This is the fix of the test.

Reply:
This relates to the previous chain of comments explaining that it's bad to mix formatting and logic changes together.